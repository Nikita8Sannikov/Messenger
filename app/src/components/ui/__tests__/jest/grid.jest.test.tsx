import { render, screen } from "@testing-library/react";

import GridComponent from "@components/ui/grid";

describe("GridComponent", () => {
    test("should render GridComponent", () => {
        render(<GridComponent><div>Test</div></GridComponent>);
        const gridElement = screen.getByText("Test");
        expect(gridElement).toBeInTheDocument();
    });

    test("matches snapshot", () => {
        const { asFragment } = render(<GridComponent></GridComponent>);

        // Снимаем снапшот компонента
        expect(asFragment()).toMatchSnapshot();
    });
});