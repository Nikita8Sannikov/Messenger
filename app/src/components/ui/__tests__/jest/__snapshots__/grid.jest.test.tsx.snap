// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridComponent matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid2-root MuiGrid2-direction-xs-row grid-item-style css-1fzlhpv-MuiGrid2-root"
  />
</DocumentFragment>
`;
